package.json

```diff
@@ -38,8 +38,8 @@
     }
   },
   "scripts": {
-    "build": "tree-sitter generate --no-bindings",
-    "test": "tree-sitter test",
+    "build": "npx tree-sitter-cli generate --no-bindings",
+    "test": "npx tree-sitter-cli test",
```

Comment:
The tree-sitter invocations were changed to make the scripts a bit more robust. Calling tree-sitter through npx like this ensures the correct version from the dev dependencies is used, and makes the scripts independent from the local system state.

Reply:
npm prefers binaries in node_modules anyway.

Reply:
> npm prefers binaries in node_modules anyway.

Ah, that's good to know. I've reverted this change then.